Sources/ComposableArchitecture/Documentation.docc/Articles/MigratingTo1.5.md

```diff
@@ -0,0 +1,165 @@
+# Migrating to 1.5
+
+Update your code to make use of the new ``Store/scope(state:action:)-90255`` operation on ``Store``
+in order to improve the performance of your features and simplify the usage of navigation APIs.
+
+## Overview
+
+The Composable Architecture is under constant development, and we are always looking for ways to
+simplify the library, and make it more powerful. As such, we often need to deprecate certain APIs
+in favor of newer ones. We recommend people update their code as quickly as possible to the newest
+APIs, and this article contains some tips for doing so.
+
+* [Store scoping with key paths](#Store-scoping-with-key-paths)
+* [Enum-driven navigation APIs](#Enum-driven-navigation-APIs)
+
+### Store scoping with key paths
+
+Prior to version 1.5 of the Composable Architecture, one was allowed to
+``ComposableArchitecture/Store/scope(state:action:)-9iai9`` a store with any kind of closures that
+transform the parent state to the child state, and child actions into parent actions:
+
+```swift
+store.scope(
+  state: (State) -> ChildState,
+  action: (ChildAction) -> Action
+)
+```
+
+In practice you could typically use key paths for the `state` transformation since key path literals
+can be promoted to closures. That means often scoping looked something like this:
+
+```swift
+ChildView(
+  store: self.store.scope(
+    state: \.child, 
```

Comment:
Shouldn't this be: 
```suggestion
    state: \.$child, 
```

Similar instance in docs id the above is valid

Reply:
The `$` is for features using property wrappers like `PresentationState`, so I don't think it's needed for this example.